spec/pgmoon_spec.moon

```diff
@@ -37,6 +37,18 @@ describe "pgmoon with server", ->
 
     assert.same true, res
 
+  it "connects with SSL", ->
+    ssl_pg = Postgres {
+      database: DB
+      user: USER
+      host: HOST
+      ssl: true
+    }
+    assert pg\connect!
+
+    finally = ->
```

Comment:
right, the intent here is to call busted's `finally`, I got confused by the moonscript syntax...